src/serviceContext.js

```diff
@@ -0,0 +1,177 @@
+const State = require('./state.js').State;
+const crypto = require('crypto');
+const KeyJar = require('../nodeOIDCMsg/src/oicMsg/keystore/KeyJar').KeyJar;
+const assert = require('chai').assert;
+
+/**
+ * @fileoverview This class keeps information that a client needs to be
+ * able to talk to a server. Some of this information comes from
+ * configuration and some from dynamic provider info discovery or client
+ * registration. But information is also picked up during the
+ * conversation with a server.
+ */
+
+/**
+ * serviceContext
+ * @class
+ * @constructor
+ */
+class ServiceContext {
+  /**
+   * This class keeps information that a client needs to be able to talk
```

Comment:
Remove the description, as it's the same as above and only keep the @param annotations.

Reply:
Fixed